spec/services/assign_edition_status_service_spec.rb

```diff
@@ -21,7 +21,7 @@
     end
 
     it "updates last edited" do
-      travel_to(Time.current) do
+      freeze_time do
```

Comment:
I like the use of `freeze_time` here, but this change should probably to fixed up to the previous commit.

Reply:
Sorted.